Marlin/src/lcd/extui/lib/mks_ui/draw_ready_print.cpp

```diff
@@ -104,14 +104,14 @@ void disp_det_error() {
 lv_obj_t *e1, *e2, *e3, *bed;
 void mks_disp_test() {
   char buf[30] = {0};
-  sprintf_P(buf, PSTR("e1:%d"), (int)thermalManager.degHotend(0));
+  sprintf_P(buf, PSTR("e1:%d"), thermalManager.degHotend(0));
   lv_label_set_text(e1, buf);
   #if HAS_MULTI_HOTEND
-    sprintf_P(buf, PSTR("e2:%d"), (int)thermalManager.degHotend(1));
+    sprintf_P(buf, PSTR("e2:%d"), thermalManager.degHotend(1));
     lv_label_set_text(e2, buf);
   #endif
   #if HAS_HEATED_BED
-    sprintf_P(buf, PSTR("bed:%d"), (int)thermalManager.degBed());
+    sprintf_P(buf, PSTR("bed:%d"), thermalManager.degBed());
```

Comment:
you need cast to int here if the code uses `%d`... in fact, is safe to keep the cast! I think we need/should keep the cast to int.
`floats` doesn't work with `%d`